sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosConflictProperties.java

```diff
@@ -55,6 +55,24 @@ Resource getResource() {
         return this.conflict;
     }
 
+    /**
+     * Gets the conflicting resource in the Azure Cosmos DB service.
+     * @param <T> the type of the object.
+     * @param klass The returned type of conflicting resource.
+     * @return The conflicting resource.
+     */
+    public <T> T getItem(Class<T> klass) {
+        return this.conflict.getItem(klass);
+    }
+
+    /**
+     * Gets the json content of the conflict.
+     * @return The content string.
+     */
+    public String getItemAsJson() {
```

Comment:
Idea of this api is customer can use their own serializer to read conflict resource . But I am doubtful as we don't support this kind of api in item read. Thoughts ?

Reply:
Removed the api getItemAsJson

Reply:
^Agree with this. As I understand it we don't currently support customer serializer in general.